### DISCUSSION TITLE
UI search for widgets

### DISCUSSION THREAD

**Discussion by lenvanessen** [plausible/analytics#525] [Anything Else]
Hi There,

I think a great addition to the UI would be the option to search within individual widgets. We have a 13k+ pages, and without search it's pretty heard to find anything that isn't in the top 100 without spending hours.

Alternatively, export would be an option as well, which is currently only available for the total number of visitors, but no other metrics.

**Comment by metmarkosaric** [comment]
thanks @lenvanessen! both of these have been requested already and would be nice to have for sure! see [here for export](https://github.com/plausible/analytics/discussions/211) and [here for search](https://github.com/plausible/analytics/discussions/265).
